core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4006,19 +4006,14 @@ definitions:
 
   AttributeUpdateOperation:
     description: ''
-    properties:
-      Example payload for AddUpdateAttribute:
-        $ref: '#/definitions/AddUpdateAttribute'
-      Example payload for RemoveAttribute:
-        $ref: '#/definitions/RemoveAttribute'
-      Example payload for AddListMember:
-        $ref: '#/definitions/AddListMember'
-      Example payload for RemoveListMember:
-        $ref: '#/definitions/RemoveListMember'
-      Example payload for CreateAttributeEntityReferenceList:
-        $ref: '#/definitions/CreateAttributeEntityReferenceList'
-      Example payload for CreateAttributeValueList:
-        $ref: '#/definitions/CreateAttributeValueList'
+    type: array
+    items:
+      - $ref: '#/definitions/AddUpdateAttribute'
+      - $ref: '#/definitions/RemoveAttribute'
+      - $ref: '#/definitions/AddListMember'
+      - $ref: '#/definitions/RemoveListMember'
+      - $ref: '#/definitions/CreateAttributeEntityReferenceList'
+      - $ref: '#/definitions/CreateAttributeValueList'
```

Comment:
this is technically invalid swagger yaml, for 2.x (there's a way to do it in 3.x). It throws an errors of `'items' must be an object` when validating. HOWEVER, it works just fine in our swagger-ui and gives examples and doc that are better than what we have now, imho. Thoughts?

Reply:
updated in the second commit; it is now valid.